### PR TITLE
Add overview of fields in plugin meta YAML in README

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Add info in README about all fields in plugin meta YAMLs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,5 +111,35 @@ icon: https://pbs.twimg.com/profile_images/929088242456190976/xjkS2L-0_400x400.j
 url: https://github.com/ws-skeleton/che-editor-theia/releases/download/untagged-892e01b21d0145207b0f/che-editor-plugin.tar.gz
 ```
 
+### Plugin meta YAML structure:
```
Comment (sleshchenko) [inline README.md]: I think it makes sense to move this section above.

File: README.md [hunk 2]
```diff
@@ -111,5 +111,35 @@ icon: https://pbs.twimg.com/profile_images/929088242456190976/xjkS2L-0_400x400.j
...
+| latestUpdateDate | date of latest plugin update (in ISO 8601) | no<sup>2</sup> |
+| preview | a URL to devfile, to preview this plugin | no |
+| tags | a list of tags, related to this plugin | no| 
+| mediaImage | links for images showcasing the plugin | no |
+| mediaVideo | links for video showcasing the plugin| no |
+| attributes | a map of special attributes, can be used for i| no |
+
+1 - Category must be equal to one of the following: "Editor", "Debugger", "Formatter", "Language", "Linter", "Snippet", "Theme", "Other"
+
+2 - dates are not required to be present in YAML, as they're generated during Plugin Registry dockerimage build
```
Comment (sleshchenko) [inline README.md]: Are they always generated or only if fields as missing?

File: README.md [hunk 3]
```diff
@@ -111,5 +111,35 @@ icon: https://pbs.twimg.com/profile_images/929088242456190976/xjkS2L-0_400x400.j
...
+| publisher| name of plugin publisher | yes |
+| repository | URL to repository of the plugin | yes |
+| category| plugin category | yes<sup>1</sup> | 
+| firstPublicationDate | date of publishing the plugin (in ISO 8601) | no<sup>2</sup> |
+| latestUpdateDate | date of latest plugin update (in ISO 8601) | no<sup>2</sup> |
+| preview | a URL to devfile, to preview this plugin | no |
+| tags | a list of tags, related to this plugin | no| 
+| mediaImage | links for images showcasing the plugin | no |
+| mediaVideo | links for video showcasing the plugin| no |
+| attributes | a map of special attributes, can be used for i| no |
```
Comment (sleshchenko) [inline README.md]: > can be used for i

What does it mean?

File: README.md [hunk 4]
```diff
@@ -111,5 +111,35 @@ icon: https://pbs.twimg.com/profile_images/929088242456190976/xjkS2L-0_400x400.j
...
+
+|field name in YAML | description | mandatory |
+|-------|-------|-------|
+| id | plugin ID | yes |
+| version | plugin version | yes |
+| type | plugin type | yes|
+| name | plugin name | yes|
+| title| plugin title | yes |
+| description| plugin description | yes |
+| icon| URL to plugin icon | yes |
```
Comment (sleshchenko) [inline README.md]: I think it makes sense to add info that is MUST be svg

File: README.md (hunk 1 above)
Comment (mkuznyetsov) [reply README.md]: moved

File: README.md (hunk 2 above)
Comment (mkuznyetsov) [reply README.md]: added more info

File: README.md (hunk 4 above)
Comment (mkuznyetsov) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (mkuznyetsov) [reply README.md]: fixed

**Pull request merged**